packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -303,6 +331,15 @@ export default class QuanticGeneratedAnswer extends LightningElement {
     }
   };
 
+  handleToggleCollapseAnswer() {
+    if (this.state?.expanded) {
```

Comment:
This looks like a nice opportunity for a ternary, up to you if you want to change it; I just like ternary (I might be the only one 😅 )

Reply:
Modified the eslint file to allow ternary calls like that.